dlpy/network.py

```diff
@@ -160,6 +183,61 @@ def compile(self):
             self.num_params += num_weights + num_bias
         print('NOTE: Model compiled successfully.')
 
+    def to_functional_model(self, stop_layers=None):
+        '''
+        Convert a Sequential into a functional model and return the functional model.
```

Comment:
I have a question about `tensor` attribute of layers.  It seems this is only supported through composing layers through `Layer.__call__`.  Then sequential models created through `Sequential.add` would not support `to_functional_model`?

Reply:
to_functional_model() only depends on self.layers, so sequential.add() or Model.from_xxx() would append a layer into self.layers or create an instance having self.layers.

Reply:
Sequential.add would support to_functional_model. to_functional_model() only depends on self.layers. Either sequential.add() or Model.from_xxx() would append a layer into self.layers or create an instance having self.layers

Reply:
I see.  Thanks for the clarification